### DISCUSSION TITLE
token问题

### DISCUSSION THREAD

**Discussion by ok419** [BurnHair/docs#104] [技术支持]
我在网站聊天界面使用了一万五千左右token,我将所有内容粘贴到opanai官网token数量和这个基本一致，但是网站令牌额度却消耗了50万额度，我实际使用了一万五千额度，我是的单独生成了一个令牌来供聊天使用，没有其他地方使用这个令牌，请问一下网站令牌消耗的额度是openai官网的33倍吗？

**Comment by BennyThink** [comment]
从记录的日志情况来看，这个令牌你一共请求了15次，最开始的时候，提示补全分别是 6701和246，以最开始的例子来计算，gpt-4o的价格是 $5和 $15 ，
```
>>> (6701*5+246*15)/1000000
.0371950000
```

<img width="799" alt="image" src="https://github.com/BurnHair/docs/assets/14024832/6aac4a04-3256-40df-a2d0-94a91b64c3d2">

之后在这个对话的基础上，你又问了一些问题，所以提示词的数量从6000开始逐步上涨。最后一次提示 17490 补全740
```
>>> (17490*5+740*15)/1000000
.0985500000
```
<img width="799" alt="image" src="https://github.com/BurnHair/docs/assets/14024832/13ad4d3c-ddb2-48cf-bddb-33e2da837ca7">

AI 就是这个样子，它没有记忆，所有的上下文实际上是包含了前面所有的聊天记录，所以聊的越久，上下文越多，消耗也就越高。

**Reply by ok419** [reply]
哦哦终于明白啦，感谢感谢。

**Reply by EFLKumo** [reply]
> 哦哦终于明白啦，感谢感谢。

所以要用能自动开新一轮对话的客户端，比如lobechat
